spark/dl/src/test/scala/com/intel/analytics/bigdl/tensor/SparseTensorSpec.scala

```diff
@@ -99,6 +101,14 @@ class SparseTensorSpec  extends FlatSpec with Matchers {
     sTensor.storageOffset() should be (1)
   }
 
+  "resize tensor to higher dim when nElement < sum(size)" should "return right result" in {
+    val indices = Array(Array(0, 4, 5, 7, 9))
+    val values = Array.fill(5)(Random.nextFloat())
+    val sTensor = Tensor.sparse(indices, values, Array(10))
+    sTensor.resize(Array(1, 10), 5)
+    Tensor.dense(sTensor).squeeze().toArray().sum should be (values.sum)
+  }
+  
```

Comment:
style check failed with following error message:
```
error file=/var/jenkins_home/workspace/BigDL-PR-V-StyleCheck/spark/dl/src/test/scala/com/intel/analytics/bigdl/tensor/SparseTensorSpec.scala message=Whitespace at end of line line=111 column=0
```

Reply:
Sorry, it has been fixed now.